addons/website_event_sale/views/templates.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="event_confirmation" inherit_id="website_sale.confirmation">
```

Comment:
There are already some inherited templates from website_sale in website_sale_templates.xml